### PR TITLE
Taa 930

### PR DESCRIPTION
**Pull request by jferdousAl**

### CODE REVIEW

File: .kitchen.docker.yml [hunk 1]
```diff
@@ -29,4 +29,5 @@ suites:
       inspec_tests:
         - name: chef-aps-inspec-tomcat
           git: https://github.com/Alfresco/chef-aps-inspec-tomcat
+          branch: 'TAA-930'
```
Comment (SubatomicHero) [inline .kitchen.docker.yml]: Should this branch be here?

File: .kitchen.yml [hunk 2]
```diff
@@ -20,5 +20,6 @@ suites:
     verifier:
       inspec_tests:
         - name: chef-aps-inspec-tomcat
-          git: https://github.com/Alfresco/chef-aps-inspec-tomcat
+          git: https://github.com/Alfresco/chef-aps-inspec-tomcat 
+          branch: 'TAA-930'
```
Comment (SubatomicHero) [inline .kitchen.yml]: Same again with reference to branch

File: .kitchen.docker.yml (hunk 1 above)
Comment (jferdousAl) [inline .kitchen.docker.yml]: The PR for the chef-aps-inspec-tomcat is not merged yet. otherwise it will use the master branch and will fail

Comment (marcomancuso) [inline .kitchen.docker.yml]: We should not merge in develop with a specific branch, let's wait to merge chef-aps-inspec-tomcat into master

File: recipes/tomcat.rb [hunk 3]
```diff
@@ -5,12 +5,20 @@
 service_name = node['tomcat']['service']
 name = [install_name, service_name].join('-')
 
+# umask = node['tomcat']['umask']
```
Comment (marcomancuso) [inline recipes/tomcat.rb]: remove these commented lines

File: .kitchen.docker.yml (hunk 1 above)
Comment (jferdousAl) [inline .kitchen.docker.yml]: okay

File: recipes/tomcat.rb (hunk 3 above)
Comment (jferdousAl) [inline recipes/tomcat.rb]: done

**Pull request merged**